fairscale/experimental/wgit/pygit.py

```diff
@@ -10,19 +10,35 @@
 
 
 class PyGit:
+    """
+    PyGit class to initialized (or wrap if already exists) the WeiGit repo as a git repo and interact with the git repo.
+
+    Args:
+        parent_path (pathlib.Path)
+            Has to be the full path of the parent!
+        gitignore (List)
+            a list of files to be added to the .gitignore
+        name (str)
+            Name of the author of the git repo
+        email (str)
+            email address of the author of the git repo
```

Comment:
```suggestion
            email address of the author of the git repo. Optionally used if it can't be determined from user's .gitconfig.
```

Reply:
on line 45/46 below, you should try to get name/email from gitconfig first. Only when those fails, use name/email passed in as arguments.